Src/LinearSolvers/MLMG/AMReX_MLEBTensor_2D_K.H

```diff
@@ -40,12 +42,24 @@ void mlebtensor_cross_terms_fx (Box const& box, Array4<Real> const& fx,
                 int jhim = j - flag(i  ,j,0).isConnected(0,-1,0);
                 int jlop = j + flag(i-1,j,0).isConnected(0, 1,0);
                 int jlom = j - flag(i-1,j,0).isConnected(0,-1,0);
+
                 Real whi = mlebtensor_weight(jhip-jhim);
                 Real wlo = mlebtensor_weight(jlop-jlom);
-                Real dudy = (0.5*dyi) * ((vel(i  ,jhip,0,0)-vel(i  ,jhim,0,0))*whi
-                                        +(vel(i-1,jlop,0,0)-vel(i-1,jlom,0,0))*wlo);
-                Real dvdy = (0.5*dyi) * ((vel(i  ,jhip,0,1)-vel(i  ,jhim,0,1))*whi
-                                        +(vel(i-1,jlop,0,1)-vel(i-1,jlom,0,1))*wlo);
+
+                Real aux = 1.0;
+                if (i==domain.hiVect()[0]+1 and bct[2*AMREX_SPACEDIM+0]==AMREX_LO_NEUMANN){
```

Comment:
Maybe we can define `const auto dlo = amrex::lbound(domain)` and `dhi` outside the loop.  It's probably faster for CPU.